webviz_subsurface/plugins/_seismic_misfit.py

```diff
@@ -161,7 +161,7 @@ def __init__(
         webviz_settings: WebvizSettings,
         ensembles: List[str],
         attributes: List[str],
-        attribute_sim_path: str = "sim2seis/output/4d_attribute_maps/",
+        attribute_sim_path: str = "share/results/maps/",
```

Comment:
Is this a new default output path? Do we need to do any changes in `testdata` or is it ok as is?

Reply:
Yes, this is a new default output. I recently made a change in testdata webviz config file to prepare for this change so it should work as is. However, we could consider also moving the data itself from `sim2seis/output/4d_attribute_maps` to `share/results/maps/`, but it is not necessary for this to work.